subworkflows/nf-core/fastq_ena_prep_adapterremoval/nextflow.config

```diff
@@ -0,0 +1,17 @@
+// IMPORTANT: Add this configuration to your modules.config
+
+process {
+    withName: '.*:FASTQ_ENA_PREP_ADAPTERREMOVAL:ENA_PREP_ADAPTERREMOVAL' {
+        ext.args = { "--qualitybase ${meta.phred}" }
+    }
```

Comment:
Make sure all other quality/and or lenght filtering is turned off: https://ena-docs.readthedocs.io/en/latest/submit/fileprep/reads.html#fastq-format